src/docs/styles/layout/_layout.scss

```diff
@@ -15,98 +15,98 @@ body {
   display: flex;
   flex-direction: column;
   height: 100vh;
-}
 
-main {
-  // TODO: ADS-198 and ADS-297 find a better place for this
-  img {
-    width: 100%;
-    margin-top: space(3x);
-  }
-}
+  .docs-body {
+    display: grid;
+    grid-template-areas: 'nav-secondary' 'main';
+    grid-template-columns: 1fr;
+    grid-template-rows: auto 1fr;
+    flex: 1;
 
-.docs-body {
-  display: grid;
-  grid-template-areas: 'nav-secondary' 'main';
-  grid-template-columns: 1fr;
-  grid-template-rows: auto 1fr;
-  flex: 1;
+    .bsds-nav-secondary {
```

Comment:
we can un-nest things here to avoid increasing specificity. ideally our css will be flat, and we can scope elements inside of a parent class if necessary.